src/tests/JIT/Regression/JitBlue/Runtime_100437/Runtime_100437.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Loader;
+using Xunit;
+
+public class Runtime_100437
+{
+    [Fact]
+    public static int TestCollectibleEmptyArrayNotInFrozenHeap()
+    {
+        WeakReference[] wrs = new WeakReference[10];
+        for (int i = 0; i < wrs.Length; i++)
+        {
+            var alc = new MyAssemblyLoadContext();
+            var a = alc.LoadFromAssemblyPath(typeof(Runtime_100437).Assembly.Location);
+            wrs[i] = (WeakReference)a.GetType("Runtime_100437").GetMethod("Work").Invoke(null, null);
+            GC.Collect();
+        }
+
+        int result = 0;
+        foreach (var wr in wrs)
+        {
+            // This is testing that the empty array from Work(), if collected, should not have been allocated on the Frozen heap
+            // otherwise it will result in a random crash.
+            result += wr.Target?.ToString()?.GetHashCode() ?? 0;
+        }
+        return result;
```

Comment:
```suggestion
        foreach (var wr in wrs)
        {
            // This is testing that the empty array from Work(), if collected, should not have been allocated on the Frozen heap
            // otherwise it will result in a random crash.
            GC.KeepAlive(wr.Target?.ToString());
        }
```
The historic convention for int returning src\tests is that 100 indicates success and everything else is failure. So returning result like this would not work.

Reply:
Trying a fix in latest commit.